api/etcd/get.go

```diff
@@ -11,23 +11,23 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Get returns the ruleset related to the given path. By default, it returns the latest one.
-// It returns the related ruleset version if it's specified.
+// Get returns the ruleset related to the given path.
```

Comment:
ISTM there's still some logic relating to the version that could usefully be documented here.
For example, if the version is empty, it returns the latest version.